sdk/communication/Azure.Communication.CallAutomation/api/Azure.Communication.CallAutomation.netstandard2.0.cs

```diff
@@ -249,12 +248,17 @@ public partial class CallDisconnected : Azure.Communication.CallAutomation.CallA
         internal CallDisconnected() { }
         public static Azure.Communication.CallAutomation.CallDisconnected Deserialize(string content) { throw null; }
     }
+    public partial class CallIntelligenceOptions
+    {
+        public CallIntelligenceOptions(System.Uri cognitiveServicesEndpoint) { }
+        public System.Uri CognitiveServicesEndpoint { get { throw null; } set { } }
+    }
     public partial class CallInvite
     {
         public CallInvite(Azure.Communication.CommunicationUserIdentifier targetIdentity) { }
         public CallInvite(Azure.Communication.MicrosoftTeamsUserIdentifier targetIdentity) { }
         public CallInvite(Azure.Communication.PhoneNumberIdentifier targetPhoneNumberIdentity, Azure.Communication.PhoneNumberIdentifier callerIdNumber) { }
-        public Azure.Communication.CallAutomation.CustomContext CustomContext { get { throw null; } }
+        public Azure.Communication.CallAutomation.CustomCallingContext CustomContext { get { throw null; } }
```

Comment:
Need to update param name as well

Reply:
Will do in next pr, good observation, thanks!